src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -120,6 +121,7 @@ const getExpandedContent = (props: ContentProps) => {
               [styles.statsGroupWithExampleLink]:
                 !group_index && !row_index && exampleLinkText
             });
+            groupsContent.push(...groupStats);
```

Comment:
I think this could just be a simple counter like `totalGroupsContent` instead of being an array.